src/connector/src/parser/postgres.rs

```diff
@@ -139,10 +141,30 @@ pub fn postgres_row_to_owned_row(row: tokio_postgres::Row, schema: &Schema) -> O
                 DataType::Decimal => {
                     handle_data_type!(row, i, name, RustDecimal, Decimal)
                 }
+                DataType::Int256 => {
+                    // Currently in order to handle the decimal beyond RustDecimal,
+                    // we use the PgNumeric type to convert the decimal to a string.
+                    // Then we convert the string to Int256.
+                    let res = row.try_get::<_, Option<PgNumeric>>(i);
```

Comment:
It seems only upstream `numeric` can be mapped into `Int256`, am I right? It looks like a hack feature, should add some comment to make it clear.

Reply:
added